Functions.Templates/Templates/OutlookMessageWebhookDeleter-CSharp/metadata.json

```diff
@@ -1,12 +1,13 @@
 ﻿{
-  "defaultFunctionName": "OultookMessageWebhookDeleterCSharp",
-  "description": "$OultookMessageWebhookDeleterCSharp_description",
-  "name": "Outlook webhook subscription deleter",
+  "defaultFunctionName": "OutlookMessageWebhookDeleterCSharp",
+  "description": "$OutlookMessageWebhookDeleter_description",
+  "name": "Outlook message webhook subscription deleter",
```

Comment:
"webhook" lower case here, so its inconsistent. I agree with lower case, myself.

Reply:
Yes the description maps to a localizable resource, where webhook is lower case.